ops/pebble.py

```diff
@@ -1299,15 +1309,17 @@ def from_dict(cls, d: '_CheckInfoDict') -> 'CheckInfo':
             status=status,
             failures=d.get('failures', 0),
             threshold=d['threshold'],
+            change_id=ChangeID(d['change-id']) if 'change-id' in d else None,
         )
 
     def __repr__(self):
         return ('CheckInfo('
                 f'name={self.name!r}, '
-                f'level={self.level!r}, '
+                f'level={self.level}, '
```

Comment:
This seems like it's better if level is a `CheckLevel` and worse if it's a `str`. Maybe both should be explicitly handled?

Reply:
True. However, realistically this is always going to be a `CheckLevel` (UP or DOWN). I think I'll leave this as is for now, as explicitly handling both is messy. We can update them all if we want in a later PR. But right now we don't test `repr()` and don't guarantee that `eval(repr(x))` works, so I think it's probably fine as is.